src/App.tsx

```diff
@@ -39,6 +40,8 @@ export default function App() {
   const [measureFileName, setMeasureFileName] = useState<string | null>(null);
   const [patientFileName, setPatientFileName] = useState<string | null>(null);
 
+  const [results, setResults] = useState<any>({});
```

Comment:
```suggestion
  const [results, setResults] = useState<any>(null);
```

Lets have this as null to begin with and only display the json viewer when the variable is non-null (see below)